scripts/Logs/README.md

```diff
@@ -0,0 +1,27 @@
+# Fornax Bash Scripts for Automating the Deployment (On-Premise)
+
+### 1. Generate ssh key in Node-B and copy ssh key ID to Node-A & Node-C:
+       ssh-keygen -t rsa
+       ssh-copy-id (node-a IP)
+       ssh-copy-id (node-c IP)
+
+### 2. Clone the repository into Node-A, Node-B, Node-C
+       git clone https://github.com/Click2Cloud-Centaurus/fornax.git -b test
+
+### 3. Edit the IP's in all the three scripts before running:
```

Comment:
it's not a good practice to edit script with meta data. use a separate file that's not checked in to read these data in. separate the responsibility of the script with its data.